### PR TITLE
Adding puppies package for astrophyics research

### PR DESCRIPTION
**Pull request by pcubillos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/puppies/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - astropy >=3.1
+    - matplotlib-base >=1.3.1
+    - mc3 >=3.0.6
+    - numpy >=1.8.1
```
Comment (mwcraig) [inline recipes/puppies/meta.yaml:32]: You can safely drop this restriction -- conda-forge hasn't support a numpy that old in a long, long time (maybe ever).

File: recipes/puppies/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+    - scipy >=0.13.3
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - puppies
```
Comment (mwcraig) [inline recipes/puppies/meta.yaml:39]: Please add this too:

```suggestion
    - puppies
  commands:
     - pip check  
  requires:    
    - pip
```

**Review by mwcraig — CHANGES_REQUESTED** [review]
@pcubillos -- this just needs a couple minor changes to merge. Thanks for opening the PR!

**Review by carterbox — APPROVED** [review]
FYI, the default channel of your GitHub repo doesn't contain any code (which is a bit strange), all the code is in the aprilis channel.

Also, for future reference, conda-forge offers prerelease channels if you're into releasing non-finalized code. However, in this case, I don't think that's your intention.

**Comment by pcubillos** [comment]
Thanks @carterbox !
Yes, I'll merge into main as soon as I'm happy with a first final release.

**Pull request merged**